openssl/src/cms.rs

```diff
@@ -0,0 +1,63 @@
+//! CMS archive
+
+use ffi;
+use foreign_types::{ForeignType, ForeignTypeRef};
+use std::ptr;
+use error::ErrorStack;
+
+use bio::{MemBio, MemBioSlice};
+
+use x509::X509;
+use pkey::PKeyRef;
+
+use cvt;
+use cvt_p;
+
+#[cfg(not(libressl))]
```

Comment:
I'd tag the entire module with this instead of sticking it on everything in here.